build/Targets/Packages.props

```diff
@@ -128,7 +128,7 @@
     <MicrosoftWin32PrimitivesVersion>4.3.0</MicrosoftWin32PrimitivesVersion>
     <MDbgVersion>0.1.0</MDbgVersion>
     <MonoOptionsVersion>4.4.0</MonoOptionsVersion>
-    <MoqVersion>4.2.1402.2112</MoqVersion>
+    <MoqVersion>4.7.99</MoqVersion>
```

Comment:
why did this need to change

Reply:
Moq 4.2.1402.2112 does not support netcoreapp2.0. I can't actually find where it states this, but I remember the build error saying something like that, and the contents of it at that version are:

```
khyperia@ashauck-desktop moq_download % cd lib
khyperia@ashauck-desktop [blah]/lib % find
.
./net35
./net35/Moq.dll
./net35/Moq.xml
./net40
./net40/Moq.dll
./net40/Moq.xml
./sl4
./sl4/Moq.Silverlight.dll
./sl4/Moq.Silverlight.xml
```

I just upped it to the latest version currently on nuget.

Reply:
(Moq is referenced by VBCSCompilerTests)

Reply:
gotcha